packages/algorand-js/test/helpers/integration.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from "fs";
+import path from "path";
+
+/**
+ * This helper adds mocha hooks to run the tests inside one of the projects
+ * from test/integration-test-files.
+ *
+ * @param filename The base name of the folder with the integration files to use.
+ */
+export function useIntegrationFile (filename: string): void {
+  let filePath: string;
+  let prevWorkingDir: string;
+
+  before(() => {
+    filePath = getIntegrationFilePath(filename);
+    prevWorkingDir = process.cwd();
+    process.chdir(filePath);
+  });
+
+  after(() => {
+    process.chdir(prevWorkingDir);
+  });
+}
+
+export function getIntegrationFilePath (
+  filename: string
+): string {
+  const filePath = path.join(
+    __dirname,
+    "..",
+    "integration-test-files",
+    filename
+  );
+  if (!fs.existsSync(filePath)) {
+    throw new Error(`Integration files ${filename} doesn't exist`);
+  }
+
+  return fs.realpathSync(filePath);
+}
```

Comment:
I was doing something similar before. Can you please check: [https://discord.com/channels/754286650540032030/754286650540032033/790732752101572608](https://discord.com/channels/754286650540032030/754286650540032033/790732752101572608)

Reply:
yes mean fixture-project part or using algob part ?
for algob part, yes i think we can copy getProgram function in test/helpers(to remove dependency)

for integration-files, i don't think it is good idea to use a single folder for all testing.
because in future if we add more complex integration files, we can easily create a different folder and use it.

Reply:
but i noticed in integration tests we are using `import { ExecParams, SignType, TransactionType } from "@algorand-builder/algob/src/types";` this creates a dependency also

Reply:
Okay, makes sense

Reply:
I think it's ok to import types for the moment, we may export some common types in the future to another package.